projects/network_tester/src/supervisor.c

```diff
@@ -782,9 +753,6 @@ static void sup_ll_after_reset_state()
         // Set drive mode by reading dl_band_cfg
         s_is_drive_mode = sup_drive_mode_init();
 
-        // Set DL Band config to default
-        sup_dl_band_config(s_is_drive_mode);
-
         // update display with module info
```

Comment:
We do not want to set the dl band to the default, we want the user's localization settings to be saved. `s_is_drive` is used by the supervisor to know when the network tester is in drive mode (shouldn't be removed). The network tester should still be configured with drive mode in the correct localization when the network tester is booting.